### PR TITLE
Fix Sonar lints

### PR DESCRIPTION
**Pull request by fperot74**

### CODE REVIEW

File: database/dbase.go [hunk 1]
```diff
@@ -109,8 +109,8 @@ func ConfigureDbDefault(v cs.Configuration, prefix, envUser, envPasswd string) {
 	v.SetDefault(prefix+"-migration-version", "")
 	v.SetDefault(prefix+"-connection-check", true)
 
-	v.BindEnv(prefix+"-username", envUser)
-	v.BindEnv(prefix+"-password", envPasswd)
+	_ = v.BindEnv(prefix+"-username", envUser)
```
Comment (harture) [inline database/dbase.go:112]: ?

File: database/dbase.go [hunk 2]
```diff
@@ -109,8 +109,8 @@ func ConfigureDbDefault(v cs.Configuration, prefix, envUser, envPasswd string) {
...
+	_ = v.BindEnv(prefix+"-password", envPasswd)
```
Comment (harture) [inline database/dbase.go:113]: ?

File: database/dbase.go [hunk 3]
```diff
@@ -280,7 +280,7 @@ func (rcdb *ReconnectableCloudtrustDB) checkError(err error) {
 			return
 		}
 		if connection.Ping() != nil {
-			rcdb.resetConnection(true)
+			_ = rcdb.resetConnection(true)
```
Comment (harture) [inline database/dbase.go:283]: ?

File: database/dbase.go [hunk 4]
```diff
@@ -344,7 +344,7 @@ func (rcdb *ReconnectableCloudtrustDB) Ping() error {
 	}
 	err = dbConn.Ping()
 	if err != nil {
-		rcdb.resetConnection(true)
+		_ = rcdb.resetConnection(true)
```
Comment (harture) [inline database/dbase.go:347]: ?

File: http/handlers.go [hunk 5]
```diff
@@ -65,7 +65,7 @@ func writeJSON(jsonableResponse interface{}, w http.ResponseWriter, statusCode i
 	var json, err = json.MarshalIndent(jsonableResponse, "", " ")
 
 	if err == nil {
-		w.Write(json)
+		_, _ = w.Write(json)
```
Comment (harture) [inline http/handlers.go:68]: ?

File: http/handlers.go [hunk 6]
```diff
@@ -98,7 +98,7 @@ func DecodeRequestWithHeaders(_ context.Context, req *http.Request, pathParams m
 	request["host"] = req.Host
 
 	buf := new(bytes.Buffer)
-	buf.ReadFrom(req.Body)
+	_, _ = buf.ReadFrom(req.Body)
```
Comment (harture) [inline http/handlers.go:101]: ?

File: http/handlers.go [hunk 7]
```diff
@@ -163,11 +163,11 @@ func ErrorHandler(logger log.Logger) func(context.Context, error, http.ResponseW
 		switch e := errors.Cause(err).(type) {
 		case security.ForbiddenError:
 			w.WriteHeader(http.StatusForbidden)
-			w.Write([]byte(errorhandler.GetEmitter() + "." + errorhandler.MsgErrOpNotPermitted))
+			_, _ = w.Write([]byte(errorhandler.GetEmitter() + "." + errorhandler.MsgErrOpNotPermitted))
```
Comment (harture) [inline http/handlers.go:166]: ?

File: http/handlers.go [hunk 8]
```diff
@@ -163,11 +163,11 @@ func ErrorHandler(logger log.Logger) func(context.Context, error, http.ResponseW
...
 		case errorhandler.Error:
 			w.WriteHeader(e.Status)
 			// You should really take care of what you are sending here : e.Message should not leak any sensitive information
-			w.Write([]byte(e.Message))
+			_, _ = w.Write([]byte(e.Message))
```
Comment (harture) [inline http/handlers.go:170]: ?

File: http/rights.go [hunk 9]
```diff
@@ -9,6 +9,6 @@ import (
 // MakeRightsHandler makes a HTTP handler that returns information about the rights of the user.
 func MakeRightsHandler(authorizationManager security.AuthorizationManager) http.HandlerFunc {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		EncodeReply(r.Context(), w, authorizationManager.GetRightsOfCurrentUser(r.Context()))
+		_ = EncodeReply(r.Context(), w, authorizationManager.GetRightsOfCurrentUser(r.Context()))
```
Comment (harture) [inline http/rights.go:12]: ?

File: http/version.go [hunk 10]
```diff
@@ -21,6 +21,6 @@ func MakeVersionHandler(componentName, ComponentID, version, environment, gitCom
 			Env:     environment,
 			Commit:  gitCommit,
 		}
-		EncodeReply(context.TODO(), w, info)
+		_ = EncodeReply(context.TODO(), w, info)
```
Comment (harture) [inline http/version.go:24]: ?

File: middleware/middleware.go [hunk 11]
```diff
@@ -96,5 +96,5 @@ func httpErrorHandler(_ context.Context, statusCode int, err error, w http.Respo
 	w.WriteHeader(statusCode)
 	w.Header().Set("Content-Type", "application/json; charset=utf-8")
 
-	w.Write([]byte(errorhandler.GetEmitter() + "." + err.Error()))
+	_, _ = w.Write([]byte(errorhandler.GetEmitter() + "." + err.Error()))
```
Comment (harture) [inline middleware/middleware.go:99]: ?

File: database/dbase.go [hunk 12]
```diff
@@ -172,7 +172,7 @@ func (cfg *DbConfig) OpenDatabase() (sqltypes.CloudtrustDB, error) {
 		}
 		err = cfg.checkMigrationVersion(dbConn)
 		if err != nil {
-			dbConn.Close()
+			_ = dbConn.Close()
```
[the remark this answers is not in the document]
Comment (fperot74) [reply database/dbase.go:175]: Not needed in this case, we are already returning en error

File: database/dbase.go (hunk 2 above)
Comment (fperot74) [reply database/dbase.go:113]: Reverted to process it correctly when we'll have time

File: database/dbase.go (hunk 3 above)
Comment (fperot74) [reply database/dbase.go:283]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: database/dbase.go (hunk 4 above)
Comment (fperot74) [reply database/dbase.go:347]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: healthcheck/health.go [hunk 13]
```diff
@@ -134,6 +134,6 @@ func (hc *healthchecker) MakeHandler() http.HandlerFunc {
 		if !status.Healthy {
 			response.StatusCode = http.StatusServiceUnavailable
 		}
-		commonhttp.EncodeReply(context.TODO(), w, response)
+		_ = commonhttp.EncodeReply(context.TODO(), w, response)
```
[the remark this answers is not in the document]
Comment (fperot74) [reply healthcheck/health.go:137]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: http/handlers.go [hunk 14]
```diff
@@ -50,7 +50,7 @@ func (r *GenericResponse) WriteResponse(w http.ResponseWriter) {
 	// Body
 	if r.MimeContent != nil {
 		w.WriteHeader(r.StatusCode)
-		w.Write(r.MimeContent.Content)
+		_, _ = w.Write(r.MimeContent.Content)
```
[the remark this answers is not in the document]
Comment (fperot74) [reply http/handlers.go:53]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: http/handlers.go (hunk 5 above)
Comment (fperot74) [reply http/handlers.go:68]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: http/handlers.go (hunk 7 above)
Comment (fperot74) [reply http/handlers.go:166]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: http/handlers.go (hunk 6 above)
Comment (fperot74) [reply http/handlers.go:101]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: http/handlers.go (hunk 8 above)
Comment (fperot74) [reply http/handlers.go:170]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: http/rights.go (hunk 9 above)
Comment (fperot74) [reply http/rights.go:12]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: http/version.go (hunk 10 above)
Comment (fperot74) [reply http/version.go:24]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: middleware/middleware.go (hunk 11 above)
Comment (fperot74) [reply middleware/middleware.go:99]: Reverted to keep sonar code smell and process it correctly when we'll have time

File: database/dbase.go (hunk 1 above)
Comment (fperot74) [reply database/dbase.go:112]: Reverted to keep sonar code smell and process it correctly when we'll have time

**Pull request merged**